.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,30 @@
 Fixes #
 
-#### Changes proposed in this Pull Request:
+### Changes proposed in this Pull Request
 
 *
 
-#### Testing instructions:
+### Testing instructions
+
+*
+
+<!-- Add the following only if there are new/updated actions or filters. Please provide a brief description of what they do and any arguments they may take. Be sure to also add the "Hooks" label to this PR. -->
+### New/Updated Hooks
+
+*
+
+<!-- Add the following only if there is any code that is being deprecated. Please list the replacement function or hook that should be called instead, if applicable. Be sure to also add the "Deprecation" label to this PR. -->
+### Deprecated Code
 
 *
 
 <!--
 Helpful tips for screenshots:
 https://en.support.wordpress.com/make-a-screenshot/
 -->
-#### Screenshot / Video
+### Screenshot / Video
 
 
 
-<!-- Add the following only if this is meant to be in changelog -->
-#### Proposed changelog entry for your changes:
+<!-- Add the following only if this is meant to be in the changelog -->
+### Proposed changelog entry for your changes
```

Comment:
Can we remove this from all templates? We can use descriptive titles in pull quests and use those as the change log entries.

Reply:
Sure! I'm doing it for all the repos ;)